### DISCUSSION TITLE
How to allow non developers to edit content outside localhost?

### DISCUSSION THREAD

**Discussion by jacktan165** [strapi/strapi#6036] [Legacy Topics]
I'm new to this.
Recently I deployed Strapi to Heroku, but Strapi disabled content builder on production mode.

I am making a website for my friend, and I want him to be able to modify his website content without calling me. I don't want to expose my localhost.

What is the best practice for this?

**Comment by robertpiosik** [comment]
Your friend should setup project locally on his machine, do necessary changes, and commit them to the repository.

Correctly configured Heroku would pull from your repository then and restart itself to reflect updated schema in connected database.

**Reply by jacktan165** [reply]
Hmm...
This is not what I expected. I guess I have to give my friend my Heroku password too.
Thanks though.

**Reply by jacktan165** [reply]
Coming back to this...
The purpose of a CMS is to allow non-developers to edit content seamlessly without any coding knowledge.
This doesn't sound right, I have to teach them how to login using Heroku CLI, tell them to install `git` and `yarn` on their machine for this to work?

**Reply by robertpiosik** [reply]
Look, `Content-Types builder` is a tool to model a database structure and it was intended to cut the power in this field for non-developers. 
> The purpose of a CMS is to allow non-developers to edit content seamlessly without any coding knowledge.

Indeed! With strapi non-developers have the interface for CRUD operations on the database (creating entries inside collection types for instance), and developers have the interface for modelling a database structures.

**Reply by jacktan165** [reply]
Oh!
Omg, I forgot the /admin at the end hahahaha!
Everything makes sense now.
Ignore everything I said, I am an idiot.

Thanks a'lot, sorry haha.
